AdvancedGenerator/Generators/NaphthaGenerator.cs

```diff
@@ -1,24 +1,53 @@
 ﻿using TUNING;
 using UnityEngine;
-using static AdvancedGeneratos.Common.GeneratorCommonConstants;
-using static AdvancedGeneratos.Generators.NaphthaGenerator;
+using static AdvancedGenerators.Common.GeneratorCommonConstants;
 
-namespace AdvancedGeneratos
+namespace AdvancedGenerators.Generators
 {
-    public class NaphthaGeneratorConfig : IBuildingConfig
+    // ReSharper disable once ClassNeverInstantiated.Global
+    public class NaphthaGenerator : IBuildingConfig
     {
+        public const string Id = nameof(NaphthaGenerator);
+
+        private const string AnimationString = "generatorpetrol_kanim";
+
+        private const int HeatSelf = 8;
```

Comment:
hard coded is yuck, put it in ctor at least? idk how mods for this work but i hope there is configuration